Build error on debug-auto-label

I'm using GHC 9.12.3. 
`async` cannot be built with the `debug-auto-label` flag:

```
% cabal install async --flags=debug-auto-label
Resolving dependencies...
Starting     async-2.2.6
Building     async-2.2.6
Failed to install async-2.2.6
Build log ( /Users/kazu/.cabal/logs/ghc-9.12.3/async-2.2.6-IEmJySVxyjcLDw01xdjZzF.log ):
cabal: Entering directory '/var/folders/k0/548g5xg90jjfbrj5j09nvwv80000gt/T/cabal-tmp-17695/async-2.2.6'
Configuring async-2.2.6...
Preprocessing library for async-2.2.6...
Building library for async-2.2.6...
[1 of 4] Compiling Control.Concurrent.Async.Internal ( Control/Concurrent/Async/Internal.hs, dist/build/Control/Concurrent/Async/Internal.o, dist/build/Control/Concurrent/Async/Internal.dyn_o )
Control/Concurrent/Async/Internal.hs:961:23: error: [GHC-88464]
    Variable not in scope: labelThread :: ThreadId -> String -> IO ()
    Suggested fixes:
      • Perhaps use ‘labelThread#’ (imported from GHC.Exts)
      • Add ‘labelThread’ to the import list in the import of ‘GHC.Conc’
        (at Control/Concurrent/Async/Internal.hs:58:1-30).
    |
961 |   myThreadId >>= flip labelThread (GHC.Stack.prettyCallStack callStack)
    |                       ^^^^^^^^^^^

Control/Concurrent/Async/Internal.hs:961:62: error: [GHC-88464]
    Variable not in scope: callStack :: GHC.Stack.CallStack
    Suggested fix:
      Perhaps use ‘GHC.Stack.callStack’ (imported from GHC.Stack)
    |
961 |   myThreadId >>= flip labelThread (GHC.Stack.prettyCallStack callStack)
    |                                                              ^^^^^^^^^

cabal: Leaving directory '/var/folders/k0/548g5xg90jjfbrj5j09nvwv80000gt/T/cabal-tmp-17695/async-2.2.6'
```

--- a/Control/Concurrent/Async/Internal.hs
+++ b/Control/Concurrent/Async/Internal.hs
@@ -55,7 +55,7 @@ import Data.IORef
 
 import GHC.Exts
 import GHC.IO hiding (finally, onException)
-import GHC.Conc (ThreadId(..))
+import GHC.Conc (ThreadId(..), labelThread)
 
 #if defined(__MHS__)
 import Data.Traversable
@@ -958,7 +958,7 @@ debugLabelMe ::
   IO ()
 debugLabelMe =
 #ifdef DEBUG_AUTO_LABEL
-  myThreadId >>= flip labelThread (GHC.Stack.prettyCallStack callStack)
+  myThreadId >>= flip labelThread (GHC.Stack.prettyCallStack GHC.Stack.callStack)
 #else
   pure ()
 #endif

